.changeset/young-roses-tickle.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/starlight': patch
+---
+
+Updates the table of contents highlighting styles to prevent UI shifts when scrolling through a page
```

Comment:
I can’t decide if this is a good idea or not 😅 

We sometimes provide migration advice in case someone is keen to preserve previous behaviour, so I guess in theory we could here too? Don’t know what you think — I guess we don’t really _want_ people to do this, but could still be handy if someone does.

````suggestion
Updates the table of contents highlighting styles to prevent UI shifts when scrolling through a page.

If you want to preserve the previous, buggy styling, you can add the following custom CSS to your site:

```css
starlight-toc a[aria-current='true'],
starlight-toc a[aria-current='true']:hover,
starlight-toc a[aria-current='true']:focus {
	font-weight: 600;
	color: var(--sl-color-text-invert);
	background-color: var(--sl-color-text-accent);
}
```
````